helprequest/views.py

```diff
@@ -73,3 +86,24 @@ def form_valid(self, form):
         # form.send_email()
         return super().form_valid(form)
 
+
+@method_decorator(login_required, name='dispatch')
+class HelpResponse(FormView):
+    template_name = 'help_response.html'
+    form_class = HelpResponseForm
+    print(form_class)
+    success_url = 'success'
+def gethelp(request):
```

Comment:
What's the difference between this function and the HelpResponse class view?

Reply:
come to think of it, no difference